web/internal-web-core-runtime/src/jsMain/kotlin/org/jetbrains/compose/web/renderComposable.kt

```diff
@@ -1,21 +1,31 @@
+/*
+ * Copyright 2020-2022 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
 package org.jetbrains.compose.web
 
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.Composition
 import androidx.compose.runtime.ControlledComposition
-import androidx.compose.runtime.MonotonicFrameClock
 import androidx.compose.runtime.DefaultMonotonicFrameClock
 import androidx.compose.runtime.DisposableEffectScope
+import androidx.compose.runtime.MonotonicFrameClock
 import androidx.compose.runtime.Recomposer
-import org.jetbrains.compose.web.dom.DOMScope
 import kotlinx.browser.document
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.CoroutineStart
 import kotlinx.coroutines.launch
-import org.jetbrains.compose.web.internal.runtime.*
+import org.jetbrains.compose.web.dom.DOMScope
+import org.jetbrains.compose.web.internal.runtime.ComposeWebInternalApi
+import org.jetbrains.compose.web.internal.runtime.DomApplier
+import org.jetbrains.compose.web.internal.runtime.DomNodeWrapper
+import org.jetbrains.compose.web.internal.runtime.GlobalSnapshotManager
+import org.jetbrains.compose.web.internal.runtime.JsMicrotasksDispatcher
 import org.w3c.dom.Element
 import org.w3c.dom.HTMLBodyElement
 import org.w3c.dom.get
+
```

Comment:
before this PR "internal-web-core-runtime" had only declarations which were not supposed to be used by external users and they all were marked with `@ComposeWebInternalApi`.

`renderComposable ` is a public function and it's used by external users. 
Does it belong to this module?

After giving it some thought:
It looks like this function is quite independent from the rest of composable-dom-API. And it's even possible to use only this module + a different implementation of html composables. 

The only thing bothering me is the name of this module. It's not internal anymore, as it contains the API intended for everyone's use. Should we rename this module?

Another question:
Is it ensured that when users add `web-core` dependency, all the APIs from "internal-web-core-runtime" will be available without adding an explicit dependency?

Reply:
should "renderComposable" have its tests in this module now?